chowda/factories.py

```diff
@@ -0,0 +1,57 @@
+from chowda.app import engine
+from chowda.config import ENGINE_URI
+from chowda.models import MediaFile, Collection, ClamsApp
+from pydantic_factories import (
+    ModelFactory,
+    SyncPersistenceProtocol,
+    AsyncPersistenceProtocol,
+)
+from typing import TypeVar, List
+from pydantic import BaseModel
+from sqlmodel import Session
+from sqlmodel.ext.asyncio.session import AsyncSession, AsyncEngine
+
+T = TypeVar("T", bound=BaseModel)
+
+
+class SyncPersistenceHandler(SyncPersistenceProtocol[T]):
+    def save(self, data: T) -> T:
+        with Session(engine) as session:
+            session.add(data)
+            session.commit()
+
+    def save_many(self, data: List[T]) -> List[T]:
+        with Session(engine) as session:
+            for d in data:
+                session.add(d)
+            session.commit()
+
+
+class AsyncPersistenceHandler(AsyncPersistenceProtocol[T]):
+    async def save(self, data: T) -> T:
+        async with AsyncSession(engine) as session:
+            session.add(data)
+            await session.commit()
+
+    async def save_many(self, data: List[T]) -> List[T]:
+        async with AsyncSession(engine) as session:
+            for d in data:
+                session.add(d)
+            await session.commit()
```

Comment:
Shamelessly copied from https://github.com/tiangolo/sqlmodel/issues/161#issuecomment-1001178233

Unfortunately, running it gives me:
```
sqlalchemy.exc.InvalidRequestError: The asyncio extension requires an async driver to be used. The loaded 'pysqlite' is not async.
```

But I think that's enough proof it works, so we should include it and we'll test it when we swap out for an async driver.